plypatch/__init__.py

```diff
@@ -384,7 +385,9 @@ def restore(self, three_way_merge=True, commit_msg=None,
         template = None
 
         # Determine whether we need to prompt the user to edit commit message
-        if commit_msg:
+        if no_commit:
```

Comment:
See above for the rename suggestion of no_commit. After that, how about we skip the pass with:

```
if commit_msg and not skip_commit:
```

Reply:
Yeah, I was avoiding rewording the "else" at the end, but I can just change it to "elif not skip_commit" I guess...